src/content/self-hosted/administration/configuration.mdx

```diff
@@ -568,12 +577,21 @@ installer:
       memory: 50Mi
 ```
 
-- `buildCredentialHelpers`: Mapping of registry hostnames as keys to credential helpers as values. Disabled if `privateRegistry` is defined. Registry hostnames must be Fully Qualified Domain Names. Credential helper names must be accessible at installer job pod `$PATH` as `docker-credential-$NAME`.
+#### buildCredentialHelpers
+
+Configure IAM roles to pull private images from your Dockerfiles. Disabled if [privateRegistry](self-hosted/administration/configuration.mdx#privateregistry) is defined.
+
+For private ECRs, add a mapping of registry hostnames as keys to credential helpers as values. Registry hostnames must be Fully Qualified Domain Names.
 
 ```yaml
 installer:
   buildCredentialHelpers:
     111122223333.dkr.ecr.eu-central-1.amazonaws.com: ecr-login
+```
+
+Create an IAM role with access to your private ECR, assume this role by the audience of Okteto Developer Service Accounts (service accounts in the `okteto` namespace) and apply these changes to the Okteto configuration:
```

Comment:
We need to explain how to do this. Is not trivial and it's not that standard yet.